homeassistant/components/system_bridge/coordinator.py

```diff
@@ -188,7 +188,10 @@ async def _setup_websocket(self) -> None:
                     session=async_get_clientsession(self.hass),
                 )
 
-            self.hass.async_create_task(self._listen_for_data())
```

Comment:
Probably an existing problem, but does anything cancel this when the integration unloads/reloads?

Reply:
I think `close_websocket` on unsub might trigger the handled exception and complete the task

Reply:
It would be nicer if it was explicitly canceled instead but that's for another PR and another day